src/SFA.DAS.Funding.SystemAcceptanceTests/Configurator.cs

```diff
@@ -20,7 +20,8 @@ private static IConfigurationRoot GetIConfigurationRoot()
             return new ConfigurationBuilder()
                 .AddEnvironmentVariables()
                 .SetBasePath(Directory.GetCurrentDirectory())
-                .AddJsonFile("local.settings.json", optional: true)
+                .AddJsonFile("appsettings.ProjectConfig.json", optional: true)
+                .AddUserSecrets("FundingTestExecutionSecrets")
```

Comment:
is this needed?

Reply:
@ruslanss Yes, we are going to read the config from the secrets moving forward for local execution, as we are now introducing some sensitive data (like connection keys)